.github/workflows/acceptance_test_dfa.yaml

```diff
@@ -0,0 +1,31 @@
+name: Deferred Actions
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "helm/**/*.go"
+      - "go.mod"
+  workflow_dispatch:
+    inputs:
+      terraformVersion:
+        description: Terraform version
+        default: 1.9.0-alpha20240516
+
+jobs:
+  acceptance_tests:
+    runs-on: custom-linux-medium
```

Comment:
is their a specific reason why the runner is on `custom-linux-medium`? just curious.

Reply:
https://github.com/hashicorp/terraform-provider-kubernetes/pull/2492/files